contrib/debian/netdata.postinst

```diff
@@ -37,16 +20,17 @@ case "$1" in
       dpkg-statoverride --update --add netdata adm 02750 /var/log/netdata
     fi
 
+    if ! dpkg-statoverride --list /usr/share/netdata/www > /dev/null 2>&1; then
+      dpkg-statoverride --update --add root netdata 0755 /usr/share/netdata/www
+    fi
+
     dpkg-statoverride --force --update --add root netdata 0775 /var/lib/netdata/registry > /dev/null 2>&1
 
     grep /usr/libexec/netdata /var/lib/dpkg/info/netdata.list | xargs -n 30 chown root:netdata
 
-    chmod 4750 /usr/libexec/netdata/plugins.d/ndsudo
-    chmod 4750 /usr/libexec/netdata/plugins.d/cgroup-network
-    chmod 4750 /usr/libexec/netdata/plugins.d/local-listeners
-
-    # Workaround for other plugins not installed directly by this package
-    chmod -f 4750 /usr/libexec/netdata/plugins.d/ioping || true
+    for f in ndsudo cgroup-network local-listeners ioping; do
+        chmod 4750 "/usr/libexec/netdata/plugins.d/${f}" || true
```

Comment:
recheck that,  ndsudo for install is not there, it's under `/usr/sbin`
```
drwxr-xr-x 3 root netdata     4096 Dec 13 07:19 .
drwxr-xr-x 5 root netdata     4096 Dec 13 07:18 ..
-rwxr-xr-x 1 root netdata     4873 Dec 12 15:41 acl.sh
-rwxr-xr-x 1 root netdata      154 Dec 12 15:41 alarm-email.sh
-rwxr-xr-x 1 root netdata   137833 Dec 12 15:41 alarm-notify.sh
-rwxr-xr-x 1 root netdata      301 Dec 12 15:41 alarm-test.sh
-rwxr-xr-x 1 root netdata     2143 Dec 12 15:41 alarm.sh
-rwxr-xr-x 1 root netdata     9122 Dec 12 15:41 anonymous-statistics.sh
-rwxr-x--- 1 root netdata   979696 Dec 12 15:41 apps.plugin
-rwxr-xr-x 1 root netdata    23650 Dec 12 15:41 cgroup-name.sh
-rwsr-x--- 1 root netdata   889520 Dec 12 15:41 cgroup-network
-rwxr-xr-x 1 root netdata    10316 Dec 12 15:41 cgroup-network-helper.sh
-rwxr-xr-x 1 root netdata     1259 Dec 12 16:41 charts.d.dryrun-helper.sh
-rwxr-xr-x 1 root netdata    21900 Dec 12 16:41 charts.d.plugin
-rwxr-x--- 1 root netdata   894672 Dec 12 15:41 debugfs.plugin
drwxr-xr-x 2 root netdata    69632 Dec 13 07:19 ebpf.d
-rwsr-x--- 1 root netdata  2296560 Dec 12 15:41 ebpf.plugin
-rwxr-xr-x 1 root netdata     1871 Dec 12 15:41 ebpf_thread_function.sh
-rwxr-xr-x 1 root netdata     2139 Dec 12 15:41 get-kubernetes-labels.sh
-rwxr-x--- 1 root netdata 64009592 Dec 12 15:41 go.d.plugin
-rwxr-xr-x 1 root netdata     7040 Dec 12 15:41 health-cmdapi-test.sh
-rwxr-xr-x 1 root netdata     6707 Dec 12 15:41 ioping.plugin
-rwsr-x--- 1 root netdata   885168 Dec 12 15:41 local-listeners
-rwxr-x--- 1 root netdata  2074320 Dec 12 15:41 logs-management.plugin
-rwxr-xr-x 1 root netdata     6860 Dec 12 15:41 loopsleepms.sh.inc
-rwsr-x--- 1 root netdata   890064 Dec 12 15:41 nfacct.plugin
-rwxr-x--- 1 root netdata   895696 Dec 12 15:41 perf.plugin
-rwxr-xr-x 1 root netdata    27950 Dec 12 16:41 python.d.plugin
-rwxr-xr-x 1 root netdata    11020 Dec 12 15:41 request.sh
-rwxr-x--- 1 root netdata   885200 Dec 12 15:41 slabinfo.plugin
-rwxr-xr-x 1 root netdata    22267 Dec 12 15:41 system-info.sh
-rwxr-x--- 1 root netdata  1178288 Dec 12 15:41 systemd-journal.plugin
-rwxr-xr-x 1 root netdata     8634 Dec 12 15:41 tc-qos-helper.sh
-rwxr-xr-x 1 root netdata     2903 Dec 12 15:41 template_dim.sh
```

Reply:
I highly suspect that that’s a bug in the build system, not an issue here. AFAIK it _was_ under `/usr/libexec/netdata` (where it arguably should be, because it’s not really meant to be used by anything _but_ netdata) previously.

Reply:
@tkatsoulas, deb12: `ndsudo` in the plugins directory. I'm not sure how it could be in a different directory, see [CMake file](https://github.com/netdata/netdata/blob/65579feafe19d9f97108ec012eac5be79235e32e/CMakeLists.txt#L1919-L1921).